helm/private/runner/runner.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+
+	"github.com/abrisco/rules_helm/helm/private/stamp"
+)
+
+func main() {
+	helm_bin := os.Getenv("HELM_BIN")
+	stable_status_file := os.Getenv("STABLE_STATUS_FILE")
+	volatile_status_file := os.Getenv("VOLATILE_STATUS_FILE")
+
+	stamps, err := stamp.LoadStamps(volatile_status_file, stable_status_file)
+	if err != nil {
+		log.Fatalf("Error loading stamps: %v", err)
+	}
+
+	var stampedArgs = []string{}
+	for _, arg := range os.Args[1:] {
+		stampedArg, err := stamp.ReplaceKeyValues(arg, stamps)
+		if err != nil {
+			log.Fatalf("Error replacing key values for %s: %v", arg, err)
+		}
+
+		stampedArgs = append(stampedArgs, stampedArg)
+	}
+
+	cmd := exec.Command(helm_bin, stampedArgs...)
```

Comment:
Will this stream logs to stderr/stdout or need to wait for the command to finish to see what's going on?

Reply:
Updated to redirect stderr and stdout directly to `os.Stdout`/`os.Stderr`.